pkgs/development/python-modules/django-dynamic-preferences/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildPythonPackage, fetchPypi
+, django, persisting-theory, six
+}:
+
+buildPythonPackage rec {
+  pname = "django-dynamic-preferences";
+  version = "1.7.1";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1z2dndkpypk4pvb0byh5a771vgp50vn8g1rbk5r3sml6dm4wcn7s";
+  };
+  buildInputs = [ six django persisting-theory ]; 
```

Comment:
These should all be `propagatedBuildInputs`
* https://github.com/EliotBerriot/django-dynamic-preferences/blob/develop/setup.py#L39